R/join_ald_scenario.R

```diff
@@ -86,6 +96,33 @@ check_portfolio_ald_scenario <- function(valid_matches, ald, scenario) {
   invisible(valid_matches)
 }
 
+add_green_technologies_to_ald <- function(data, scenario) {
+
+  green_techs <- r2dii.data::green_or_brown %>%
+    filter(.data$green_or_brown == "green") %>%
```

Comment:
non-blocking: avoid conflicts with `base::filter` but explicitly namespacing `dplyr::filter`

Reply:
Good shout. We (mauro and I) elected to import most frequently used `dplyr` verbs in the `imports.R` file, to help readability, but filter does seem like a special case that we should revisit. 

That said, we are not namespacing filter in just about every function in this package, so I would elect to handle this in a dedicated PR, rather than adding it to this one.

Reply:
See #334